attributes/cacher.rb

```diff
@@ -0,0 +1,2 @@
+default['ssh_known_hosts']['cacher']['data_bag'] = 'server_data'
+default['ssh_known_hosts']['cacher']['data_bag_item'] = 'known_hosts'
```

Comment:
Never experienced the problem of switching between files, as I have a file list to the left with sublime.
I will put it back to the default.rb if you have such strong feelings about this.